core/src/test/scala/com/pingcap/tispark/datasource/BaseDataSourceTest.scala

```diff
@@ -34,19 +38,27 @@ class BaseDataSourceTest(val table: String,
 
   protected def dropTable(): Unit = jdbcUpdate(s"drop table if exists $dbtable")
 
-  protected def tidbWrite(rows: List[Row],
-                          schema: StructType,
-                          param: Option[Map[String, String]] = None): Unit = {
+  protected def dropTable(tblName: String): Unit =
+    jdbcUpdate(s"drop table if exists $database.$tblName")
```

Comment:
`
```suggestion
    jdbcUpdate(s"drop table if exists $dbtable")
```

Reply:
Nope. I was using a different table name.

Reply:
add ` before and after $database and $tblName

Reply:
ok